drivers/pwm/pwm_nrf5_sw.c

```diff
@@ -7,11 +7,22 @@
 #include <soc.h>
 
 #include "pwm.h"
+#include <nrf_peripherals.h>
 
 #define LOG_LEVEL CONFIG_PWM_LOG_LEVEL
 #include <logging/log.h>
 LOG_MODULE_REGISTER(pwm_nrf5_sw);
 
+/* One compare channel is needed to set the PWM period, hence +1. */
+#define PWM_0_MAP_SIZE DT_NORDIC_NRF_SW_PWM_0_CHANNEL_COUNT
+#if ((DT_NORDIC_NRF_SW_PWM_0_CHANNEL_COUNT + 1) > \
+     (_CONCAT( \
+        _CONCAT(TIMER, DT_NORDIC_NRF_SW_PWM_0_TIMER_INSTANCE), \
+        _CC_NUM)) \
+     )
```

Comment:
Either indent this line one space less or move the parenthesis to the previous line (this I'd prefer, and I guess it will be more consistent with Zephyr coding style). Now it is more misleading than helping.

Reply:
Moved up